### PR TITLE
Testing with ember-cli-page-object v2.0.0-beta.3

### PR DESCRIPTION
**Pull request by DanMonroe**
This PR is intended our efforts to get `ember-table` compatible with Ember v4.

Changes include:

- Uses the `v2.0.0-beta3` branch of `ember-cli-page-object`  https://github.com/san650/ember-cli-page-object/releases
- Removes ` useNativeEvents`
- Drops Ember 2.8 and 2.12 ember-try scenarios

Perhaps we could release a beta version of this addon while we wait for a release version of v2 of `ember-cli-page-object`

### CODE REVIEW

**Review by kpfefferle — APPROVED** [review]
I'd suggest that we merge this to `master` and release it as an `0.8.0-beta` for the ember-table `4.0-beta` branch to use? Then when ember-cli-page-object releases a 2.0 stable, we can upgrade to that, release an `0.8` stable here, and then leverage that in the new version of ember-table?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Node Modules Cache
         uses: actions/cache@v2
         with:
-          path: '**/node_modules'
+          path: "**/node_modules"
```
Comment (twokul) [inline .github/workflows/ci.yml:23]: It seems we use single quotes in this file. Why the change?

**Review by twokul — APPROVED** [review]
One small comment, but lgtm overally!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (DanMonroe) [reply .github/workflows/ci.yml]: reverted.

**Comment by twokul** [comment]
Going to publish beta today
